whd/dispatcher.py

```diff
@@ -175,6 +175,11 @@ def _matching_resources(self, concourse_api, event):
             if not resource.type == resource_type:
                 continue
             ghs = resource.github_source()
+            if any(skip in event.commit_message() for skip in ('[skip ci]', '[ci skip]')):
+                if not ghs.disable_ci_skip():
+                    continue
+                if ghs.disable_ci_skip() in ('False', 'false'):
```

Comment:
use `yaml.safe_load`. preferrable do the parsing in the getter method and return a boolean value from it (this is what the model classes should do..)